source/_components/homematic.markdown

```diff
@@ -277,6 +278,25 @@ To get the `homematic.keypress` event for some HomeMatic IP devices like WRC2 /
 8. When your channel is working now, you can edit it to select the other channels one by one
 9. At the end, you can delete this program from the CCU
 
+#### {% linkable_title Low battery event %}
+
+As described above devices with buttons are not implemented as actual entities inside Home Assistant. Therefore you cannot watch a low battery state on these.
+To solve this for all homematic IP devices (including those devices with buttons) the event **homematic.low_bat** is fired when battery is running low. you can then use this in your automations:
```

Comment:
Capital _H_, and actually I think this isn't limited to just IP devices. The regular wireless devices with batteries should behave the same way. 🤔

Reply:
Darn. Sloppy coder I am... I can't check on the regular wireless devices. Shall I remove "IP" ?

Reply:
I'm 99% sure it will work. Can't test right now, but if it doesn't then I'll take care of the fix.